src/server/rfm/rfm_funcs/create_bins.py

```diff
@@ -0,0 +1,54 @@
+def create_bins(data, query_date):
+    '''This script will take table data and bin edges for RFM scores for all PAWS donations
+
+    query_date = date data was queried
+    '''
+
+    import pandas as pd
+    import numpy as np
+    import jenkspy
+    from datetime import datetime, date
+    import os
+
+
+
+    ####
+    # read in csv
+    donations_df = pd.read_csv('csv')
+
+    donations_df['Close_Date'] =pd.to_datetime(donations_2021['Close_Date']).dt.date
+
+    ##################################################################################
+    # Calculate recency bins
+    from recency_bins import recency_bins
+    recency_bins, quantile_scores= recency_bins(donations_df, query_date)
+
+    ###################################################################################
+    # Calculate frequency bins
+    from frequency_bins import frequency_bins
+
+    jenks_frequency_bins, human_frequency_bins = frequency_bins(donations_df)
+
+
+        def checkIfDuplicates(listOfElems):
+        ''' Check if given list contains any duplicates '''
+        for elem in listOfElems:
+            if listOfElems.count(elem) > 1:
+                return True
+        return False
+
+        duplicats_bool = checkIfDuplicates(jenks_frequency_bins)
+        if duplicates_bool  == True:
+            final_frequency_bins = human_frequency_bins

```

Comment:
Are these supposed to be the same variable? If not, maybe change one to be less similar.